nautobot/core/templates/generic/object_list.html

```diff
@@ -20,14 +55,47 @@
     {% endif %}
 </div>
 <h1>{% block title %}{{ content_type.model_class|meta:"verbose_name_plural"|bettertitle }}{% endblock %}</h1>
+{% if filter_params %}
+<div class="filters-applied">
+    <b>Filters:</b>
+    {% for field, values in filter_params %}
+    <span class="filter-container display-inline" dir="ltr">
```

Comment:
Why do we need two separate classes here? Can we remove the `display-inline` class and just apply all necessary CSS to `filter-container`?